### PR TITLE
refactor: log format

### PR DESCRIPTION
**Pull request by sivchari**
Issue #, if available:

*Description of changes:*

I fixed log format.
It's failed on purpose to confirm the logs.

```
DEBU[0000] Creating limactl command: ARGUMENTS: [ls -f {{.Status}} finch], LIMA_HOME: /user/workspace/finch/lima/data 
DEBU[0000] Status of virtual machine:                   
INFO[0000] dependency socket_vmnet file not found: open /user/workspace/finch/dependencies/lima-socket_vmnet/opt/finch/bin/socket_vmnet: no such file or directory 
INFO[0000] Requesting root access to finish network dependency configuration 
Password:
DEBU[0004] Creating limactl command: ARGUMENTS: [sudoers], LIMA_HOME: /user/workspace/finch/lima/data 
DEBU[0004] config file not found: %!w(*fs.PathError=&{open /user/workspace/finch/lima/data/_config/override.yaml 2}) 
INFO[0004] dependency socket_vmnet file not found: open /user/workspace/finch/dependencies/lima-socket_vmnet/opt/finch/bin/socket_vmnet: no such file or directory 
ERRO[0004] Dependency error: failed to install dependencies: [Failed to finish installing rootful dependencies which are needed for external network access within the guest OS. Boot will continue, but container exposed ports will not be accessible from macOS.: [error copying files to directory /opt/finch, err: exit status 1, stderr: cp: /user/workspace/finch/dependencies/lima-socket_vmnet/opt/finch: No such file or directory
 skipping installation of network configuration because pre-requisites are missing]] 
FATA[0004] failed to load the lima config file: open /user/workspace/finch/lima/data/_config/override.yaml: no such file or directory
```
logrus finally call [(*Entry).Logf](https://github.com/sirupsen/logrus/blob/master/entry.go#L347). The function uses fmt.Sprintf to format. So, %w cannot use to format.

*Testing done:*

- [x] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kokes** [comment]
Note that this duplicates my earlier PR: https://github.com/runfinch/finch/pull/39

File: cmd/virtual_machine_init.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (iva *initVMAction) run() error {
 
 	err = dependency.InstallOptionalDeps(iva.optionalDepGroups, iva.logger)
 	if err != nil {
-		iva.logger.Error(fmt.Sprintf("Dependency error: %s", err))
+		iva.logger.Errorf("Dependency error: %s", err.Error())
```
Comment (davidhsingyuchen) [inline cmd/virtual_machine_init.go:71]: Could we use `%v` and `err` instead of `%s` and `err.Error()` to be consistent with other logging statements?

**Review by davidhsingyuchen — COMMENTED** [review]
Thanks for the contribution! LGTM overall. Could you merge the latest `main` to incorporate the changes in #39?

File: cmd/virtual_machine_init.go (hunk 1 above)
Comment (sivchari) [reply cmd/virtual_machine_init.go]: Thanks ! I fixed.

File: pkg/dependency/vmnet/update_override_lima_config.go [hunk 2]
```diff
@@ -92,7 +92,7 @@ func (overConf *overrideLimaConfig) appendNetworkConfiguration(filePath string)
 	}
 	defer func() {
 		if err := f.Close(); err != nil {
-			overConf.l.Errorf("error closing file at path %s, error: %v", filePath, err)
+			overConf.l.Errorf("error closing file at path %s, error: %s", filePath, err.Error())
```
Comment (davidhsingyuchen) [inline pkg/dependency/vmnet/update_override_lima_config.go:95]: Just one missing place: https://github.com/runfinch/finch/pull/48#discussion_r1031964747. Thanks!

Comment (sivchari) [reply pkg/dependency/vmnet/update_override_lima_config.go]: Sorry, I fixed. Thanks!

**Review by davidhsingyuchen — COMMENTED** [review]
Could you help fix the [failing unit tests](https://github.com/runfinch/finch/actions/runs/3545625402/jobs/5953911594)? You can run `make test-unit` locally to verify them. Thanks!

**Comment by davidhsingyuchen** [comment]
Changing the PR title from `fix:` to `refactor` due to https://github.com/runfinch/finch/pull/48#issuecomment-1326492033.

**Pull request closed without merging**